src/main/java/no/nav/fo/veilarbregistrering/registrering/resources/RegistreringTilstandDto.java

```diff
@@ -0,0 +1,18 @@
+package no.nav.fo.veilarbregistrering.registrering.resources;
+
+import no.nav.fo.veilarbregistrering.registrering.bruker.Status;
+
+public class RegistreringTilstandDto {
+
+    private long id;
```

Comment:
Jeg lurte på å bruke UUID i stedet for database-id - at vi logger UUID og bruker den mer funksjonelt, mens id bare blir en teknisk greie

Reply:
Hva er fordelen med det?

Reply:
Tenker at UUID er mer funksjonelt å bruke i APIer enn databasenøkler - ønsker ikke å dele databaseId mer enn nødvendig på utsiden